Homebrew/emacs_formula.rb

```diff
@@ -81,7 +81,7 @@ def ert_run_tests(*files)
 
   ### Slightly better tested than ert_run_tests
   def byte_compile(*files)
-    emacs_args = %W[ --batch -Q ]
```

Comment:
That's odd.  There's nothing in the manpage, but I did have to add `-Q` in addition to `-batch` for running the js2 tests: https://github.com/mooz/js2-mode/pull/348